src/equations/ideal_glm_mhd_2d.jl

```diff
@@ -1118,6 +1118,20 @@ end
     return p
 end
 
+# Transformation from conservative variables u to d(p)/d(u)
+@inline function gradient_u(::typeof(pressure),
+                            u, equations::IdealGlmMhdEquations2D)
```

Comment:
@ranocha Is this name ok for you or do you have a better idea?

Reply:
I think the name `gradient_u` is a bit unusual in our current context since it includes the symbol of the conservative variables instead of the meaning of them. For example, we use function names like `pressure` instead of `p`, `density` instead of `rho` etc. Thus, the name `gradient_conservative` seems to fit better, I think. It would also be more consistent with the name `GradientVariablesConservative` used in the parabolic context.

Reply:
That's a very good point - it is also more descriptive than my suggestion. @bennibolm would you mind fixing this up accordingly?

Reply:
Done in [3cd1c95](https://github.com/trixi-framework/Trixi.jl/pull/1738/commits/3cd1c9508c4b79f064c4991ace89d678c99539f5).